src/GitHub.Exports/Exports/ExportMetadata.cs

```diff
@@ -42,4 +44,12 @@ public interface IViewModelMetadata
     {
         UIViewType ViewType { get; }
     }
+
+    public static class ExportViewAttributeExtensions
+    {
+        public static bool IsViewType(this UserControl c, UIViewType type)
+        {
+            return c.GetType().GetCustomAttributesData().Any(x => x.AttributeType.Equals(typeof(ExportViewAttribute)) && (UIViewType)x.NamedArguments[0].TypedValue.Value == type);
```

Comment:
If it's an ExportViewAttribute, it has to have a named argument list with one entry, it's part of the class